PlaywrightSolar/PageObjects/SolarShopPage.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.Playwright;
+
+namespace PlaywrightSolar.PageObjects;
+
+public class SolarShopPage(IPage page)
+{
+    private const string SolarShopPageUrl = "https://solartechnology.com.ua/shop";
+    
+    // Locators
+    private ILocator BasketIcon => page.Locator(".cart-icon");
+   
+    // Methods
+    public async Task VerifyBasketIsEmpty()
+    {
+        await page.WaitForURLAsync(SolarShopPageUrl);
+
+        await Assertions.Expect(BasketIcon).ToBeVisibleAsync();
+        
+        await Assertions.Expect(BasketIcon).ToHaveAttributeAsync("class", "cart-icon ");
```

Comment:
this is extra. BasketIcon was found by CSS locator ".cart-icon" which means "attribute 'class' contains 'cart-icon', so no need to check that element which was found by class "cart-icon" has class "cart-icon"